src/common/components/cards/instance-details-footer.tsx

```diff
@@ -27,5 +31,24 @@ export const InstanceDetailsFooter = NamedFC<InstanceDetailsFooterProps>('Instan
         return null;
     }
 
-    return <div>Highlight {highlightState}</div>;
+    const kebabMenuIcon = () => {
+        return <CardKebabMenuButton />;
+    };
+
+    const HighlightButton = () => {
+        const label = 'HighLight Buttons ' + highlightState;
```

Comment:
Should we also change the variable and label here to not have "button" in them?

Reply:
Not sure but this is out of scope of this PR, I used what was there already, we can check with Dan later and fix it in a different PR if we need to